### PR TITLE
Remove mentioning of subscription message sending

### PR DESCRIPTION
**Pull request by danielmarbach**

### CODE REVIEW

**Comment by danielmarbach** [comment]
@Particular/nservicebus-maintainers please review

File: nservicebus/messaging/publish-subscribe/controlling-what-is-subscribed.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ reviewed: 2016-11-07
 
 ## Automatic subscriptions
 
-The default mode for managing subscriptions is *auto subscribe*. When a subscriber endpoint is started, it determines  which events it needs to subscribe to. The endpoint then sends subscription messages to the publisher endpoint for those messages. This auto subscription mechanism happens each time the subscriber is restarted.
+The default mode for managing subscriptions is *auto subscribe*. When a subscriber endpoint is started, it determines  which events it needs to subscribe to. This auto subscription mechanism happens each time the subscriber is restarted. For more information how publish and subscribe works refer to the [Publish-Suscribe](/nservicebus/messaging/publish-subscribe) documentation page.
```
Comment (bording) [inline nservicebus/messaging/publish-subscribe/controlling-what-is-subscribed.md]: Revision:

The default mode for managing subscriptions is *auto subscribe*.  Every time a subscriber endpoint is started, it determines which events it needs to subscribe to and automatically subscribes to them. For more information on how publish and subscribe works, refer to [Publish-Suscribe](/nservicebus/messaging/publish-subscribe).

**Comment by danielmarbach** [comment]
Thanks @bording

**Comment by danielmarbach** [comment]
I'll merge once green

**Pull request merged**